presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -60,7 +60,7 @@ statement
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
     | SHOW GRANTS
-        ON (ALL | TABLE? qualifiedName)                                #showGrants
```

Comment:
@martint I am unclear on how namespaces will be resolved when there are two entities named xxx and someone issues "SHOW GRANTS ON xxx". Perhaps we can defer resolving this issue until we add support for grants on entities other than tables.